src/test/java/io/stargate/sgv2/jsonapi/service/operation/tables/DeleteUpdateWhereAnalyzerTest.java

```diff
@@ -301,4 +304,27 @@ public void skip1and3of3ClusteringKey(
         .eqOnlyOneClusteringKey(1)
         .analyzeMaybeFilterError(expectedCode);
   }
+
+  private static Stream<Arguments> inFilterUsage() {
+    return Stream.of(
+        Arguments.of(StatementType.DELETE_ONE, InTableFilter.Operator.IN),
+        Arguments.of(StatementType.DELETE_ONE, InTableFilter.Operator.NIN),
+        Arguments.of(StatementType.UPDATE_ONE, InTableFilter.Operator.IN),
+        Arguments.of(StatementType.UPDATE_ONE, InTableFilter.Operator.NIN));
+  }
+
+  @ParameterizedTest
+  @MethodSource("inFilterUsage")
+  public void skip1and3of3ClusteringKey(
+      StatementType statementType, InTableFilter.Operator inTableFilterOperator) {
+
+    var fixture =
+        TEST_DATA
+            .whereAnalyzer()
+            .table2PK3Clustering1Index("in_filter_on_" + statementType.name(), statementType);
+    fixture
+        .expression()
+        .inOnOnePartitionKey(inTableFilterOperator)
+        .analyzeMaybeFilterError(FilterException.Code.INVALID_IN_FILTER_FOR_UPDATE_ONE_DELETE_ONE);
```

Comment:
Can this also check the message ? see the comment for the error.yaml, we can check the message contains - replace the var

The command used an invalid filter on the columns: ${inFilterColumns}.

Reply:
fixed